Slambook/index.js

```diff
@@ -1,14 +1,26 @@
 let form;
 
 let allSlamArray=[];
+let formValue = {};
+let err ;
 //Write code below to select the form element
 form = document.querySelector('form')
 console.log(form,"form");
 // Write your code above
 
 // Add the evenlistner below
 form.addEventListener('submit',function(e){
+    
+    formValue={}
+    allSlamArray=[];
     e.preventDefault();
+    if (err){
+    err.forEach((element)=>{
+        let tag = element.parentNode.querySelector('input')
+        element.remove(element)
+        tag.style = ""
+    });
+    }
```

Comment:
Consider adding comments or properly organizing the code for better readability and maintainability.